ariadne_codegen/utils.py

```diff
@@ -70,3 +71,13 @@ def format_multiline_strings(source: str) -> str:
             formatted = convert_to_multiline_string(orginal_str, variable_indent_size)
             formatted_source = formatted_source.replace(orginal_str, formatted)
     return formatted_source
+
+
+def process_name(name: str, convert_to_snake_case: bool) -> str:
+    """Processes the GraphQL name to remove keywords and optionally convert to snake_case."""
```

Comment:
This line now is too long 😅 `ariadne_codegen/utils.py:77:0: C0301: Line too long (93/88) (line-too-long)`